editor/editor_properties.cpp

```diff
@@ -2953,7 +2953,7 @@ bool EditorInspectorDefaultPlugin::parse_property(Object *p_object, Variant::Typ
 
 			} else {
 				EditorPropertyFloat *editor = memnew(EditorPropertyFloat);
-				double min = -65535, max = 65535, step = 0.001;
+				double min = -65535, max = 65535, step = 0.000001;
```

Comment:
That will make all floats have a high precision in the inspector, I'm not sure this is a good change.

Reply:
This should probably go in another PR as it will require more discussion that the rest of the changes, and doesn't seem blocking to add these VS constants.

Reply:
Okay, returned it back to old precision